src/components/editor/rubric-criteria-groups-editor.js

```diff
@@ -291,8 +284,13 @@ class RubricCriteriaGroupsEditor extends composeMixins(
 			return;
 		}
 
+		const criteriaGroupPadding = styleMap({
+			paddingBottom: criteriaGroupIndex === this._criteriaGroupCount - 1 ? 0 : '24px'
```

Comment:
```suggestion
			paddingBottom: criteriaGroupIndex === this._criteriaGroupCount - 1 ? '0' : '24px'
```
Not sure if it matters functionality wise, but this should be a string